pkg/logutil/logging_test.go

```diff
@@ -50,6 +52,41 @@ func newFile(j int) *backuppb.File {
 	}
 }
 
+type isAbout struct{}
+
+func (isAbout) Info() *CheckerInfo {
+	return &CheckerInfo{
+		Name: "isAbout",
+		Params: []string{
+			"actual",
+			"expect",
+		},
+	}
+}
+
+func (isAbout) Check(params []interface{}, names []string) (result bool, error string) {
+	actual := params[0].(float64)
+	expect := params[1].(float64)
+
+	if diff := math.Abs(1 - (actual / expect)); diff > 0.1 {
+		return false, fmt.Sprintf("The diff(%.2f) between actual(%.2f) and expect(%.2f) is too huge.", diff, actual, expect)
+	}
+	return true, ""
+}
+
+func (s *testLoggingSuite) TestRater(c *C) {
```

Comment:
Will this become a unstable test...

Reply:
Maybe we should make the diff even higher(say, 0.2 or 0.5)... Or when fail, just send a warn🤔？

Reply:
Well, a `RateAt` function can help us. PTAL.
```go
timePass := time.Now()
rater.Success(1)
c.Assert(rater.RateAt(timePass.Add(100*time.Millisecond), time.Second), isAbout{}, 10.0)
rater.Success(1)
c.Assert(rater.RateAt(timePass.Add(150*time.Millisecond), time.Second), isAbout{}, 13.0)
rater.Success(18)
c.Assert(rater.RateAt(timePass.Add(200*time.Millisecond), time.Second), isAbout{}, 100.0)
```